honeybee_energy/hvac/allair/_base.py

```diff
@@ -63,12 +63,13 @@ class _AllAirBase(_TemplateSystem):
         * latent_heat_recovery
         * demand_controlled_ventilation
         * schedules
+        * user_data
     """
     __slots__ = ('_economizer_type', '_sensible_heat_recovery', '_latent_heat_recovery',
-                 '_demand_controlled_ventilation')
+                 '_demand_controlled_ventilation', '_user_data')
     ECONOMIZER_TYPES = ('NoEconomizer', 'DifferentialDryBulb', 'DifferentialEnthalpy',
                         'DifferentialDryBulbAndEnthalpy', 'FixedDryBulb',
-                        'FixedEnthalpy', 'ElectronicEnthalpy')
+                        'FixedEnthalpy', 'ElectronicEnthalpy', '_user_data')
```

Comment:
This one looks like you copy/pasted things into the wrong tuple. Just remove this one and it should be good.

Reply:
Oops, see commit: "wrong touple"